doc-2759: start of VMO Appliance docs (#10329)
spectrocloud/librarium: docs/docs-content/vm-management/vmo-appliance/install-vmla-iso.md

| -------------------- | --------------------------------------------- | -------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------- |
| **CPU**              | Intel or AMD64 CPU with 8 cores               | Intel or AMD64 CPU with 8 cores                    | -                                                                                                                     |
| **RAM**              | 24 GB                                         | 256 GB or more                                     | Assumes the deployment of 20 VMs per node multiplied by the median RAM per VM.                                        |
| **Network Adapters** | 2 x 1 Gbps (data + management)                | 2 x 10 Gbps (data) <br /> 2 x 10 Gbps (management) | Pod overlay operates on the management network.                                                                       |

> | **Storage Adapters** | 2 x 16 Gbps FC                                | 2 x 16 Gbps FC                                     | Storage adapters must support the Fibre Channel (FC) protocol, a high-speed network protocol used for data transfer.  |

Reviewer: ```suggestion
| **Storage Adapters** | 2 x 16 Gbps FC or 2 x 10Gb Ethernet shared with data + management                          | 2 x 16 Gbps FC or 2 x 10Gb Ethernet dedicated                                    | Dedicated Storage Adapters, either Fiber Channel or Ethernet (eg iSCSI) will provide reliable access to external or cluster shared storage  |
```

| -------------------- | --------------------------------------------------------------------- | -------------------------------------------------- | ---------------------------------------------------------------------------------------------------- |
| **CPU**              | Intel or AMD x64 CPU with 8 cores                                     | Intel or AMD x64 CPU with 8 cores                  |                                                                                                      |
| **RAM**              | 24 GB                                                                 | 256 GB or more                                     | Assumes the deployment of 20 VMs per node multiplied by the median RAM per VM.                       |
| **Network Adapters** | 2 x 1 Gbps <br /> (data + management)                                 | 2 x 10 Gbps (data) <br /> 2 x 10 Gbps (management) | Pod overlay operates on the management network.                                                      |

> | **Storage Adapters** | 2 x 16 Gbps FC                                                        | 2 x 16 Gbps FC                                     | Storage adapters must support the FC protocol, a high-speed network protocol used for data transfer. |

Reviewer: ```suggestion
| **Storage Adapters** | 2 x 16 Gbps FC or 2 x 10Gb Ethernet shared with data + management                          | 2 x 16 Gbps FC or 2 x 10Gb Ethernet dedicated                                    | Dedicated Storage Adapters, either Fiber Channel or Ethernet (eg iSCSI) will provide reliable access to external or cluster shared storage  |
```

6.  Fill out the **Profile Config** page and select **Next**.

7.  Fill out the following fields for each section.

    ### Networking

    | **Parameter**                        | **Description**                                                                                                                        |
    | ------------------------------------ | -------------------------------------------------------------------------------------------------------------------------------------- |
    | **Pod CIDR**                         | IP address range assigned to internal Kubernetes pod networking. Change only if this conflicts with your existing network.             |
    | **Service CIDR**                     | IP address range reserved for Kubernetes services, such as internal load balancers and DNS. Must not overlap with Pod Network Range.   |
    | **MetalLB IP Address**               | A single unused IP address on your network that exposes cluster services externally.                                                   |

>     | **Cilium and MetalLB interface**     | The physical network interface on each node used for cluster traffic and external service announcements.                               |

Reviewer: ```suggestion
    | **Cilium and MetalLB interface**     | The physical network interface, bond, or bridge on each node used for cluster traffic and external service announcements.                               |
```

    | **Enable VLAN Filtering (Optional)** | When enabled, the bridge interface permits only VLANs listed in **VLAN range for VMs**. Disable unless you need strict VLAN isolation. |
    | **VLAN range for VMs**               | VLAN IDs that tenant VMs can use. Accepts individual IDs, such as `12` and `13`, or ranges, such as `15-20`.                           |
    | **Bridge Interface**                 | The Linux bridge interface on cluster nodes that connects tenant VMs to the physical network. Leave blank to auto-detect.              |
    | **Cluster runs on br0**              | Enable if your Kubernetes cluster nodes communicate via the br0 bridge interface or a VLAN sub-interface of br0.                       |

>     | **VLANs on top of br0**              | List all VLAN IDs configured as sub-interfaces on `br0`. Include VLAN 1. For example, `1,10,20`.                                       |

Reviewer: ```suggestion
    | **VLANs on top of br0**              | List all VLAN IDs configured as sub-interfaces or to be dynamically attached on `br0`. Include VLAN 1 and all VM VLANs also. For example, `1,10,20`.                                       |
```

    ### Local Admin

    | **Parameter**                 | **Description**                                                                                     |
    | ----------------------------- | --------------------------------------------------------------------------------------------------- |
    | **VMO Local Admin User Name** | Username for the local fallback administrator account used when OIDC authentication is unavailable. |
    | **VMO Local Admin Password**  | Password for the local fallback administrator account. This value is stored securely.               |

    ### Storage

    | **Parameter**                                 | **Description**                                                                                                                                            |
    | --------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |

>     | **Storage Node Interface**                    | The network interface on each node dedicated to storage replication traffic between nodes. Choose a high-bandwidth interface when possible.                |

Reviewer: ```suggestion
    | **Storage Node Interface**                    | The network interface on each node dedicated to storage replication traffic between nodes. Choose a high-bandwidth interface or bond when possible.                |
```

   | **VMs by Namespace**         | Breakdown of VMs by running, stopped, and other statuses.                                                                 |
   | **VM Needing Attention**     | List of unhealthy VMs.                                                                                                    |

### Auto-Refresh and Pause

The dashboard polls the API and metrics backend on a configurable interval (5 seconds, 15 seconds, or 30 seconds). Use
the **interval selector** in the toolbar to change the cadence. Select **Pause** to stop all background polling, which
is useful when inspecting data or troubleshooting. Select **Resume** to restart polling.

### Customize the Layout

You can customize the interface by dragging widget headers to reorder widgets within the grid, resizing widgets from
their bottom-right corner handle, and adding or removing widgets with the **+** button in the toolbar. Select **Reset
Layout** to return all widgets to the default arrangement. Layout changes save automatically and persist across
sessions.

## Next Steps

> 

Reviewer 2: The first step in the create your first VM is to upload ISO but there are steps in the getting started section that should be done first. While the getting started guide points to the things that need to be set up, they are not explained. 

<img width="349" height="715" alt="Image" src="https://github.com/user-attachments/assets/fa8f1c1a-212e-4741-85b7-74445140f7c7" />

Author: @kylejep Please see this page: https://deploy-preview-10329--docs-spectrocloud.netlify.app/vm-management/vmo-appliance/getting-started-wiz/

Barring any style guide changes, I'm thinking this goes out as is when we do release and I'll be filling in details/options more after. Would that work?
